bzl/plaidml.bzl

```diff
@@ -141,3 +154,178 @@ plaidml_settings = rule(
     },
     implementation = _plaidml_settings_impl,
 )
+
+
+def _plaidml_target(settings, attr):
+    return {"//plaidml:target": attr.plaidml_target}
+
+
+plaidml_target = transition(
```

Comment:
Can you add some comments as to what this is and what it's doing?

Reply:
Done

Reply:
What are we transitioning from and to?

Reply:
We're transitioning `//plaidml:target`.

So, for example:

* `bazel build --//plaidml:target=llvm_cpu //plaidml/edsl/tests:cc_test` -- builds `cc_test` for `llvm_cpu`

* `bazel build --//plaidml:target=intel_gen //plaidml/edsl/tests:cc_test` -- builds `cc_test` for `intel_gen`
* `bazel build --//plaidml:target=llvm_cpu //plaidml:intel_gen_test` -- builds `//plaidml/edsl/tests:cc_test` for `intel_gen` and builds `bazel-bin/plaidml/intel_gen.tar.gz` from it.  This happens regardless of what you specify for `--//plaidml:target` on the command line.